HealthcareAnalytics/HealthcareAnalytics/Views/Shared/_Layout.cshtml

```diff
@@ -25,9 +25,19 @@
                     <li>@Html.ActionLink("Home", "Index", "Home")</li>
                     <li>@Html.ActionLink("Patients", "Index", "Patients")</li>
                     <li>@Html.ActionLink("Employees", "Index", "Employees")</li>
+                    <li class="dropdown">
+                        <a href="#" data-toggle="dropdown" class="dropdown-toggle">Incidents<b class="caret"></b></a>
+                        <ul class="dropdown-menu">
+                            <li><a href="Incidents/General/">General Incident</a></li>
```

Comment:
Do you think this should be hard coded? We have a table of incident types. We can iterate through each item in that table instead.